docs/pages/load-testing/results.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Test results
+sidebar_label: Results
+---
+
+## Summary
+
```

Comment:
This document includes performance test results of Kubernetes API using various vCluster and K8s distributions and configurations.